src/main/scala/com/etsy/conjecture/scalding/util/DynamicOptions.scala

```diff
@@ -0,0 +1,127 @@
+package com.etsy.conjecture.scalding.util
+
```

Comment:
I thought if I used Args everywhere I would have to do type checks and conversations everywhere and it might be messier?  I wanted a something to hold the Args in their actual types that I could also maybe in the future add more operations too.  I'm also just used to having an options class inside of a toolkit, seemed like a better long term solution for more complex things in the future.  But that might be a preference thing.